### PR TITLE
fix(admin/mission): export const dynamic = 'force-dynamic';

### PR DESCRIPTION
**Pull request by yenting-biao**

### CODE REVIEW

**Comment by yenting-biao** [comment]
DO NOT MERGE IT NOW!

**Comment by yenting-biao** [comment]
Two update:
1. `export const dynamic = 'force-dynamic';` Ref: <https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamic>. I am sure that it is correct.
2. `important: true` in tailwind config file. Ref: <https://stackoverflow.com/a/74893538>. I am not sure what it is for, but the tailwind css looks good in the updated branch deployment link.

Summary:
You can merge it I think. If the beta version is no problem after merging, the version in wp1121 can also be updated.

**Review by togimoto — APPROVED** [review]
Let's see if this works

**Pull request merged**
